core/domain/suggestion_registry.py

```diff
@@ -61,7 +61,7 @@ class BaseSuggestionDict(TypedDict):
     target_version_at_submission: int
     status: str
     author_name: str
-    final_reviewer_id: Optional[str]
+    final_reviewer_id: str
```

Comment:
Does it make sense for this to be non-optional? When a suggestion is first submitted, there is no reviewer yet. /cc @sagangwee 

(Ditto in multiple places below.)

Reply:
Reverted to previous, Optional[str]